tests/testutil/service.go

```diff
@@ -198,13 +207,15 @@ func (ts *TestService) Promote(args ...string) {
 }
 
 // Logs calls "rio logs ns/service" on this service
-func (ts *TestService) Logs(args ...string) string {
-	args = append([]string{"logs"}, append(args, ts.Name, "-a")...)
-	out, err := RioCmd(args)
+func (ts *TestService) Logs(args ...string) []string {
+	args = append(
+		[]string{"logs"},
+		append(args, ts.Name)...)
+	results, err := RioCmdWithTail(10, args)
```

Comment:
So this follow logs for 10 seconds and then kills it ? Ok for now but feels like a stopgap solution. Hopefully in next month or so we will make a log dump option instead of always following.

Reply:
Yep exactly! Attach is doing the same thing. I agree we need a logdump though!